cmd/f3/pubsub.go

```diff
@@ -0,0 +1,22 @@
+package main
```

Comment:
Renamed the file due to typo in file name and moved common pubsub utility stuff from here to internal/psutil package.